bitrise.yml

```diff
@@ -133,11 +133,18 @@ workflows:
         run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
     - restore-spm-cache@2.1.1:
         is_always_run: true
+    - script@1.2.1:
+        title: Override xctestrun file to UnitTests
+        inputs:
+        - content: |-
+            envman add --key BITRISE_TEST_SHARDS_PATH_FIREFOX --value ""
```

Comment:
we need the values of this variables, they can not be always empty only when build or test steps fail